CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Install new grid on each components.
```

Comment:
Tous les éléments liés au StyleGuide, nous ne les mettons pas dans le CHANGELOG.
Par contre la modification sur la taille `$button-tiny-min-width: k-px-to-rem(110px);`, c'est une modification sur un composant de `kitten` qu'il faut ajouter dans ce fichier.